### PR TITLE
DEV-5919 document monochart development

### PR DESCRIPTION
**Pull request by PiotrPalkaSpotOn**
- document monochart development 
  - mounting single file from Config Map on the pod 
  - using Aplication Load Balancer in EKS environment

- DEV-5919

### CODE REVIEW

File: monochart/README.md [hunk 1]
```diff
@@ -133,10 +132,188 @@ releases:
...
+            {{- if index .Values "vanityDomains" }}
+              {{ range $domain := .Values.vanityDomains }}
+              '{{ $domain }}':
+                {{- if index $root.Values "public_paths" }}
+                  {{- range $path := $root.Values.public_paths }}
+                - '{{ . }}'
+                  {{- end }}
+                {{- end }}
+              {{- end }}
+            {{- end }}
```
Comment (PiotrPalkaSpotOn) [inline monochart/README.md:291]: Indents looks improper but for go template it doesn't matter. I try to keep it with indents to not lose track of `{{- <function> }}` `{{ -end }}` pair. It's a compromise between readability and template values placement respectivity.

File: monochart/README.md [hunk 2]
```diff
@@ -133,10 +132,188 @@ releases:
...
+
+We can make ingress address available from public Internet but sometimes we want to limit access just to Spoton
+employees. In this case ZScaler integration comes handy. From the networking perspective we create ALB rule to allow
+access to traffic that source originates from one of ZScaler IP addresses.
+
+The general idea is that all the URL paths are behind ZScaler but we can expose some endpoints publicly.
+
+#### How to start
 
-## TODO:
```
Comment (PiotrPalkaSpotOn) [inline monochart/README.md:137]: it got moved to lower section

File: monochart/README.md
Comment (MaxymVlasov) [inline monochart/README.md:1]: That's interesting. Do we have any references to this doc in [docs.corp.spoton.sh](https://docs.corp.spoton.sh/)?

Or how anybody should know that there exist some docs?

Comment (PiotrPalkaSpotOn) [reply monochart/README.md:1]: hmm good point. I don't think there's any. We should document the whole deployment process I think and add a link to this readme

**Comment by PiotrPalkaSpotOn** [comment]
@fcooper-spoton it's very useful

**Comment by PiotrPalkaSpotOn** [comment]
@fcooper-spoton I'm forcing to merge it because it takes too long time and I'll update the https://github.com/SpotOnInc/helm-repository/blob/main/charts/README.md with this PR

**Pull request merged**